.gitignore

```diff
@@ -34,3 +34,20 @@
 
 # unless supporting rvm < 1.11.0 or doing something fancy, ignore this:
 .rvmrc
+
+*.iml
+.gradle
+/local.properties
+/.idea/caches
+/.idea/libraries
+/.idea/modules.xml
+/.idea/workspace.xml
+/.idea/navEditor.xml
+/.idea/assetWizardSettings.xml
+.DS_Store
+/build
+/captures
+.externalNativeBuild
+.cxx
+local.properties
```

Comment:
`/local.properties` and `local.properties` 🤔 And do we have the `/captures` in the generated folder? 

FYI: I often use the common gitignore for Android project from [here](https://github.com/github/gitignore/blob/master/Android.gitignore)

Reply:
for the `capture` : Those files are capture files generated when running the Layout Inspector or the Method Tracing in Android Studio. 
I got this config when create new project with new AS version. I guess it is the new feature of AS.

Reply:
@doannimble So how about the `/local.properties` and `local.properties`? I believe it's redundant.

Reply:
@hoangnguyen92dn maybe the `/local.properties` is redundant. But the other one is not

Reply:
@hoangnguyen92dn removed `/local.properties`